packages/ui/feature-dashboard/src/lib/services/folders.service.ts

```diff
@@ -0,0 +1,46 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import {
+  CreateFolderRequest,
+  Folder,
+  FoldersListDto,
+  SeekPage,
+} from '@activepieces/shared';
+import { environment } from '@activepieces/ui/common';
+import { Observable, map } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FoldersService {
+  constructor(private http: HttpClient) {}
+  create(req: CreateFolderRequest): Observable<FoldersListDto> {
+    return this.http.post<Folder>(environment.apiUrl + '/folders', req).pipe(
+      map((res) => {
+        return {
+          ...res,
+          numberOfFlows: 0,
```

Comment:
Make DTO in the backend that returns with numberOfFlows: 0

Don't hardcode here